### PR TITLE
[0.18.x] Update SPEC file to pick the latest go version

### PR DESCRIPTION
**Pull request by navidshaikh**

### CODE REVIEW

File: openshift-serverless-clients.spec [hunk 1]
```diff
@@ -4,7 +4,7 @@
 %global product_name OpenShift Serverless
 %global golang_version 1.14
 %global kn_version 0.18.4
-%global kn_release 1
+%global kn_release 2
 %global kn_cli_version v%{kn_version}
```
Comment (navidshaikh) [inline openshift-serverless-clients.spec:8]: @warrenvw : do we need to bump golang_version to specific one here?
current at line no 5: `golang_version 1.14`

Comment (warrenvw) [reply openshift-serverless-clients.spec:8]: If the specific version is required, then you can use golang_version 1.14.12

**Pull request merged**